js/main.js

```diff
@@ -0,0 +1,85 @@
+/* Here's what the pine tree should look like when you specify a height of 7, and use the asterisk character.
+
+01        *
+03       ***
+05      *****
+07     *******
+09    *********
+11   ***********
+13  *************
+
+*/
+
+var pineTree = {
+  height: 0,
+  species: "null"
+};
+
+var inputs = document.getElementsByTagName("input");
+var treeHeight = inputs[0];
+var treeSpecies = inputs[1];
+
+document.getElementsByTagName("input")[0].addEventListener("keydown", function(){
```

Comment:
Provide an `id` property for all of your HTML elements. Then use `getElementById` instead of `getElementsByTagName`. It's best to be as specific as possible with your code.